### ISSUE TITLE
Private Messages sometimes appear blank.

### ISSUE THREAD

**Issue by r-kinesis** [Sceleratis/Adonis#459]
Upon sending a pm to someone, they will see it, but at some occasions, it won't appear. Has this been recognized? 
![image](https://user-images.githubusercontent.com/84187653/129125528-abcb8216-33d2-45d1-8136-44bd0c8ddddc.png)

**Comment by Kw6m** [comment]
You can send a blank PM; are you sure this is not what they are doing (if you press send with no text it will send a blank PM)

**Comment by r-kinesis** [comment]
No, I've received reports that players did type something but they always end up blank,

**Comment by fxeP1** [comment]
> No, I've received reports that players did type something but they always end up blank. If you notice in the picture, 3 people gave me a blank pm. This has happened on multiple occasions when we're sending pms to a certain team..

Have you checked your scriptlogs and seen the reply inside it (an unfiltered result is outputted), This also could be an adonis issue with its `service.Filter` but its good you check next time it happens again if it actually shows inside scriptlogs.

**Comment by r-kinesis** [comment]
No it doesn't really show anything in scriptlogs, sadly.
